src/desktop/multipath.desktop

```diff
@@ -13,6 +13,7 @@ X-SuSE-YaST-AutoInst=
 X-SuSE-YaST-Geometry=
 X-SuSE-YaST-SortKey=
 X-SuSE-YaST-AutoInstResource=multipath
+X-SuSE-YaST-Keywords=network,tcp
```

Comment:
I am not sure about TCP, but I do not have a strong opinion here.

Reply:
Well the communication is over TCP. Also not sure but it does not harm here. :-)